package.json

```diff
@@ -893,7 +894,7 @@
     "@kbn/zod-helpers": "link:packages/kbn-zod-helpers",
     "@langchain/community": "^0.0.20",
     "@langchain/core": "^0.1.45",
-    "@langchain/openai": "^0.0.12",
+    "@langchain/openai": "^0.0.14",
```

Comment:
Since `langchain` is now on the latest, shall we bump this the latest ([`0.0.25`](https://www.npmjs.com/package/@langchain/openai)) as well?